README.md

```diff
@@ -85,7 +85,7 @@ Clone this repository to your desired folder:
 👤 **Emeka Emmanuel**
 
 - GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
```

Comment:
- [ ] Please make sure the the view button is aligend to the center as in the design 

| Your  | Behance |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/210811827-5ba057df-f540-451a-9699-5e76842d4253.png) |![image](https://user-images.githubusercontent.com/74744787/210812067-99410744-57ec-41bb-ad9c-d604d3224c4b.png) |

Reply:
✔️